src/System.ServiceModel.Security/src/System.ServiceModel.Security.csproj

```diff
@@ -10,12 +10,12 @@
   <!-- Help VS understand available configurations -->
   <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'Debug|AnyCPU'" />
   <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'Release|AnyCPU'" />
-  <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'net46_Debug|AnyCPU'" />
```

Comment:
I believe these changes are unexcercised because noone is building the net4* configuration of this project.